### PR TITLE
fixing delete workload for unsupported data services

### PR DESCRIPTION
**Pull request by bhsrinivasan-px**
**What this PR does / why we need it**:
Fix for deleting workloads, skip dataservices that do not currently run workload generation scripts

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/pds/dataservice_test.go [hunk 1]
```diff
@@ -905,12 +905,14 @@ var _ = Describe("{RestartPXPods}", func() {
 
 				defer func() {
 					Step("Delete the workload generating deployments", func() {
-						if ds.Name == "Cassandra" || ds.Name == "PostgreSQL" {
-							err = pdslib.DeleteK8sDeployments(dep.Name, namespace)
-						} else {
-							err = pdslib.DeleteK8sPods(pod.Name, namespace)
+						if !(ds.Name == mysql || ds.Name == kafka || ds.Name == zookeeper) {
```
Comment (Madanagopal19) [inline tests/pds/dataservice_test.go:908]: Fix looks good to me, can we add it all places ?

**Review by Madanagopal19 — CHANGES_REQUESTED** [review]

File: tests/pds/dataservice_test.go (hunk 1 above)
Comment (bhsrinivasan-px) [reply tests/pds/dataservice_test.go:912]: done

**Review by Madanagopal19 — APPROVED** [review]
LGTM

**Comment by Madanagopal19** [comment]
Pull the latest changes and add 
`dash = Inst().Dash`  before all dash instance

**Review by Madanagopal19 — APPROVED** [review]
LGTM

**Pull request merged**